test/integration/http_protocol_integration.cc

```diff
@@ -58,4 +58,72 @@ std::string HttpProtocolIntegrationTest::protocolTestParamsToString(
                       upstreamToString(params.param.upstream_protocol));
 }
 
+void HttpProtocolIntegrationTest::expectUpstreamBytesSentAndReceived(
```

Comment:
This change omitted H3 bytes accounting. And it could break existing monitoring. It would be great if things could have been communicated earlier, i.e. a tracking issue or cc'ing H3 code owners.